### PR TITLE
Updating to Fleet v0.3.10-rc5

### PR DESCRIPTION
**Pull request by mattfarina**

### CODE REVIEW

File: charts/fleet-agent/100.0.4+up0.3.10-rc5/Chart.yaml.orig [hunk 1]
```diff
@@ -6,8 +6,8 @@ annotations:
   catalog.cattle.io/permits-os: linux,windows
   catalog.cattle.io/release-name: fleet-agent
 apiVersion: v2
-appVersion: 0.3.10-rc4
```
Comment (aiyengar2) [inline charts/fleet-agent/100.0.4+up0.3.10-rc5/Chart.yaml.orig:9]: this file shouldn't exist? Seems like there was an error when applying the patch file in `packages/fleet/fleet-agent/generated-changes/patch`

File: charts/fleet/100.0.4+up0.3.10-rc5/Chart.yaml.orig [hunk 2]
```diff
@@ -9,12 +9,12 @@ annotations:
   catalog.cattle.io/provides-gvr: clusters.fleet.cattle.io/v1alpha1
   catalog.cattle.io/release-name: fleet
 apiVersion: v2
-appVersion: 0.3.10-rc4
```
Comment (aiyengar2) [inline charts/fleet/100.0.4+up0.3.10-rc5/Chart.yaml.orig:12]: this file also should not exist

**Review by aiyengar2 — CHANGES_REQUESTED** [review]
Need to clean up the Chart.yaml.orig? Not sure why it came up

**Review by aiyengar2 — APPROVED** [review]

**Review by jiaqiluo — APPROVED** [review]
LGTM

**Pull request merged**